torch/csrc/jit/passes/batch_mm.cpp

```diff
@@ -391,7 +391,23 @@ void BatchMMSide(Block * block, const AliasDb& alias_db) {
 
 }
 
+bool hasMutableOperators(Block* block) {
+  for (auto n : block->nodes()) {
+    if (n->kind().is_aten() && n->schema().is_mutable())
```

Comment:
Why does it matter if it's ATen or not?

Reply:
other ops might not have schema information